src/services/elasticBeanstalkApplication/schema.graphql

```diff
@@ -1,8 +1,8 @@
 type awsElasticBeanstalkApp implements awsBaseService @key(fields: "arn") {
   name: String @search(by: [hash, regexp])
   description: String @search(by: [hash, regexp])
-  elasticBeanstalkEnv: [awsElasticBeanstalkEnv] #change to plural
+  iamServiceRole: String @search(by: [hash, regexp])
+  elasticBeanstalkEnvs: [awsElasticBeanstalkEnv]
```

Comment:
does this need an update to the `field` for the connection file where this connection is?

Reply:
Yes, it was done. Or do you mean when the connection is formed?